slash/parallel/parallel_manager.py

```diff
@@ -89,16 +58,8 @@ def start_server_in_thread(self, collected):
         self.keepalive_server_thread.start()
 
     def kill_workers(self):
-        if config.root.tmux.enabled:
-            for worker_pid in self.server.worker_pids:
-                try:
-                    os.kill(worker_pid, signal.SIGTERM)
-                except OSError as err:
-                    if err.errno != errno.ESRCH:
-                        raise
-        else:
-            for worker in self.workers.values():
-                worker.send_signal(signal.SIGTERM)
+        for worker in self.workers.values():
```

Comment:
I would turn that into a list in case at some point in the future the dictionary will change to modify itself based on worker availability:
```
for worker in list(self.workers.values()):
```